cartesian_pose_controller:
  type: mars_control/CartesianPoseController
  arm_id: panda
